### DISCUSSION TITLE
Feature request: Ability to turn dependabot on/off per language

### DISCUSSION THREAD

**Discussion by caleb15** [community/community#11204] [Code Security]
A user might want to turn dependabot off for a certain language in a specific repo. For example, Javascript generates many false positives due to alerts warning about code that would be vulnerable in the backend but is fine in the frontend. Or maybe they already have security scanning in CI or elsewhere for that language.

**Comment by exvuma** [comment]
This should be possible using the `package-ecosystem` setting in your `dependabot.yml`. See the [Configuration options for dependency updates docs](https://docs.github.com/en/enterprise-server@3.3/code-security/supply-chain-security/keeping-your-dependencies-updated-automatically/configuration-options-for-dependency-updates#configuration-options-for-updates) and let me know if that gives you trouble

**Reply by Danack** [reply]
Hi @exvuma ,

Is there a setting for "about code that would be vulnerable in the backend but is fine in the frontend"? I had a look and couldn't see one.

Most of my projects are PHP in the backend and JS in the browser, so all of the JS vulnerabilities caused by prototype pollution or DOS from regexes (i.e. most JS vulnerabilities) aren't actually security threats for me, so it'd be really great to be able to not receive warnings about them.

**Comment by caleb15** [comment]
Does that work for security alerts too or just dependency updates?

**Reply by ramimac** [reply]
+1: Was about to file a separate discussion, currently documentation indicates the `dependabot.yml` configuration is only handy if you're doing updates - if it supports security alerts that will solve a lot of my problems!
